packages/client/wallets/aa/src/api/CrossmintWalletService.ts

```diff
@@ -124,4 +124,12 @@ export class CrossmintWalletService extends BaseCrossmintService {
             `Error fetching NFTs for wallet: ${address}`
         );
     }
+
+    async getWalletPolicies(address: string, chain: string) {
```

Comment:
it's hard to tell what should be a part of the SDK and what's just for us–does this function have any use for a client?
If not, we can make a subclass of `CrossmintWalletService` that we use in the test site where we add this, but it shouldn't go into the SDK

Reply:
Only will be used for testing. Moved to the 'front' package in a subclass that extends the CrossmintWalletService.